### PR TITLE
pretty print the json object while exporting to file, fix #3690

### LINKED ISSUE #3690: prettify metadata.json generated from console on export

### PR DESCRIPTION
**Pull request by codingkarthik**
### Affected components

- [ ] Server
- [X] Console
- [ ] CLI
- [ ] Docs
- [ ] Community Content
- [ ] Build System
- [ ] Tests
- [ ] Other (list it)

### Related Issues

#3690 

### Solution and Design

Added the space parameter in the JSON.stringify() function.

### Steps to test and verify

Export the metadata.json file

#### Catalog upgrade

Does this PR change Hasura Catalog version?
- [ X] No
- [ ] Yes
  - [ ] Updated docs with SQL for downgrading the catalog 

#### Metadata

Does this PR add a new Metadata feature?
- [ X] No
- [ ] Yes
  - Does `run_sql` auto manages the new metadata through schema diffing?
    - [ ] Yes
    - [ ] Not required
  - Does `run_sql` auto manages the definitions of metadata on renaming?
    - [ ] Yes
    - [ ] Not required
  - Does `export_metadata`/`replace_metadata` supports the new metadata added?
    - [ ] Yes
    - [ ] Not required

#### GraphQL
- [ X] No new GraphQL schema is generated
- [ ] New GraphQL schema is being generated:
   - [ ] New types and typenames are correlated
   
   

#### Breaking changes

- [X ] No Breaking changes
- [ ] There are breaking changes:

  1. Metadata API

     Existing `query` types:
     - [ ] Modify `args` payload which is not backward compatible
     - [ ] Behavioural change of the API
     - [ ] Change in response `JSON` schema
     - [ ] Change in error code
     

  2. GraphQL API

     Schema Generation:
     
     
     - [ ] Change in any `NamedType`
     - [ ] Change in table field names
     

     Schema Resolve:-
     
     - [ ] Change in treatment of `null` value for any input fields 
     

  3. Logging

     - [ ] Log `JSON` schema has changed
     - [ ] Log `type` names have changed

### CODE REVIEW

**Comment by rikinsk** [comment]
@codingkarthik Thanks for raising this PR so quickly! Could you please sign the CLA so we can merge this.

File: console/src/components/Common/utils/jsUtils.js [hunk 1]
```diff
@@ -225,7 +225,10 @@ export const downloadObjectAsJsonFile = (fileName, object) => {
     : fileName + jsonSuffix;
 
   const dataString =
-    'data:' + contentType + ',' + encodeURIComponent(JSON.stringify(object));
+    'data:' +
+    contentType +
+    ',' +
+    encodeURIComponent(JSON.stringify(object, null, 4));
```
Comment (rikinsk) [inline console/src/components/Common/utils/jsUtils.js]: Could you set indent size to 2 in place of 4. 4 seems a little extra spaced out

Comment (codingkarthik) [reply console/src/components/Common/utils/jsUtils.js]: Sure @rikinsk, I'll make that change

**Comment by codingkarthik** [comment]
Hey Richin,

I have made the changes and also signed the CLA.

**Pull request merged**